Functions/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -192,23 +203,23 @@
 				warning('Did not succeed in repeating PLDs for each TE for Hadamard sequence import');
```

Comment:
Note that this warning could be misleading if you have a Hadamard sequence with only one echotime!

Reply:
Just remove warning? @jan-petr